packages/backend/src/graphql/resolvers/combined_payments.ts

```diff
@@ -3,24 +3,25 @@ import {
   QueryResolvers,
   Payment as SchemaPayment
 } from '../generated/graphql'
-import { ApolloContext } from '../../app'
+import { TenantedApolloContext } from '../../app'
 import { getPageInfo } from '../../shared/pagination'
 import { Pagination, SortOrder } from '../../shared/baseModel'
 import { CombinedPayment } from '../../open_payments/payment/combined/model'
 
-export const getCombinedPayments: QueryResolvers<ApolloContext>['payments'] =
+export const getCombinedPayments: QueryResolvers<TenantedApolloContext>['payments'] =
   async (parent, args, ctx): Promise<ResolversTypes['PaymentConnection']> => {
     const combinedPaymentService = await ctx.container.use(
       'combinedPaymentService'
     )
-    const { filter, sortOrder, ...pagination } = args
+    const { filter, sortOrder, tenantId, ...pagination } = args
     const order = sortOrder === 'ASC' ? SortOrder.Asc : SortOrder.Desc
 
     const getPageFn = (pagination_: Pagination, sortOrder_?: SortOrder) =>
       combinedPaymentService.getPage({
         pagination: pagination_,
         filter,
-        sortOrder: sortOrder_
+        sortOrder: sortOrder_,
+        tenantId: ctx.isOperator ? tenantId : ctx.tenant.id
```

Comment:
It's only just occurring to me now that the resolver tests for tenanted resources don't test if the proper scope of results is returned when a `tenantId` argument is provided by an operator versus a tenant...

Reply:
Captured in https://github.com/interledger/rafiki/issues/3476.